src/react-components/room/RoomSettingsSidebarContainer.js

```diff
@@ -13,6 +15,16 @@ export function RoomSettingsSidebarContainer({ showBackButton, room, hubChannel,
     settings => {
       hubChannel.updateHub(settings);
       onClose();
+
+      NotifiablePermissions.forEach(perm => {
+        if (APP.hub.member_permissions[perm] !== settings.member_permissions[perm]) {
+          hubChannel.sendMessage({
```

Comment:
Is there a reason this isn't just part of `hubChannel.updateHub(settings)` like the other settings?

Reply:
We are not sending messages for room permissions updates. I guess we could just listen to `permission_update` events and call `hubChannel.receive` locally instead?

Reply:
We send messages for other permission updates like "allow object spawning" and "allow drawing" no? Shouldn't this be the same?

Reply:
We don't do for room permissions although if we are doing it for chat/voice probably we should also do for the rest? Maybe that could be part of a follow-up PR as this is how it's in production right now.

Reply:
Yeah seems worth changing on the server side so this can be removed as a special case.